plugins/woocommerce/client/legacy/css/twenty-nineteen.scss

```diff
@@ -1203,15 +1203,17 @@ table.variations {
 			&:before {
 				content: "";
 				display: inline-block;
-				width: 16px;
-				height: 16px;
+				width: 16px !important;
+				height: 16px !important;
```

Comment:
Are there any paths you see to achieving this without `!important` (ie, increasing specificity)?